docs/products/opensearch/howto/migrate-ism-policies.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Reapply ISM policies after snapshot restore
+sidebar_label: Reapply ISM policies
+limited: true
+---
+
+Reapply Index State Management (ISM) policies to Aiven for OpenSearch® using a script.
+
+After restoring your snapshot, ISM policies that manage index rollover, retention, and
+deletion must be reapplied to your indices. These policies are stored in the
+`.opendistro-ism-config` index, but the assignments between indices and policies must
+be manually reapplied using a script.
+
+## What is restored
+
+The `.opendistro-ism-config` index stores ISM policy configurations and is restored with
+the snapshot, but policy assignments to specific indices are stored in the cluster
+metadata and must be manually reapplied..
+
+## Prerequisites
+
+- A machine with network access to Aiven for OpenSearch services
+- Python 3.11 or higher installed
+- Ensure all data indices and the `.opendistro-ism-config` index are restored from the
+  snapshot
+- The snapshot was restored with `include_global_state: true`
+
+:::warning
+
+- **Snapshot must include global state**
+  Ensure the snapshot was restored with `include_global_state: true`. If it was
+  restored without global state, the ISM policy assignments will not be available in the
+  cluster metadata, and the script will fail to reapply the policies.
+
+- **Script can only be run once**
+  OpenSearch clears the cluster metadata with ISM policy assignments after the policies
+  are applied. This means the script can only be run once. To run the script again,
+  restore the snapshot.
+
+:::
+
+## Check for active ISM policies
+
+Before executing the reapply script, confirm that no indices are currently managed by
+ISM policies. Run the following command to check:
+
+```bash
+curl -X GET --insecure -s "${AIVEN_SERVICE_URI}/_plugins/_ism/explain?pretty"
+```
+
+If the result shows `"total_managed_indices": 0`, no policies are applied, and you can
+proceed. Otherwise, remove the existing policies if necessary.
```

Comment:
This is not the recovery, e.g. removing the policies will help if the script has already been run targeting the service as the metadata it uses is no longer available.

Reply:
Thanks. Updated the text to clarify that removing policies is only needed if the script has already been run and the metadata is unavailable. Also, moved this into the next section.